tensorhive/api/api_specification.yml

```diff
@@ -37,6 +41,67 @@ paths:
           description: Duplication error.
         '500':
           $ref: '#/responses/InternalError'
+  /user/logout:
+    delete:
+      tags:
+        - users
+      summary: Logout user
+      operationId: 'tensorhive.api.controllers.user.post_logout_access_token'
+      responses:
+        '201':
+          description: Successfully deleted.
+        '500':
+          $ref: '#/responses/InternalError'
+      security:
+          - Bearer: []
+  /user/logout1:
```

Comment:
Maybe rename this endpoint to something more meaningful.

Reply:
Agreed. Actually, why is it needed?